updateSub/index.js

```diff
@@ -0,0 +1,34 @@
+'use strict'
```

Comment:
Rename this directory `api_updateSubscriber`

Azure functions doesn't currently support deploying from directories :( so we should just prefix every directory that contains an azure function with `api_` for organization